static/js/settings_users.js

```diff
@@ -114,14 +114,14 @@ function populate_users(realm_people_data) {
 
     var bots_table_html = "";
     _.each(bots, function (user) {
-        var bot_html = templates.render("admin_user_list", {user: user});
+        var bot_html = templates.render("admin_user_list", {user: user, is_loggedin_user_admin: page_params.is_admin});
         bots_table_html = bots_table_html.concat(bot_html);
     });
     bots_table.append(bots_table_html);
 
     _.each(active_users, function (user) {
         var activity_rendered;
-        var row = $(templates.render("admin_user_list", {user: user}));
+        var row = $(templates.render("admin_user_list", {user: user, is_loggedin_user_admin: page_params.is_admin}));
```

Comment:
@showell Yeah, we can. But I thought that this way it would be more clear as `admin_user_list.handlebars` begins with `{{#with user}}` that is it iterates through each user in the list and `is_admin` implies if this particular user in the list is admin or not.
https://github.com/zulip/zulip/pull/4463/commits/4ac334695e9154dc55a6b0394bff97ff3ffcd098#diff-a59b18976202f07c9a72bff359282f4bL31
So, to be clear I used `is_loggedin_user_admin`. What are your thoughts on this?
(If `is_admin` seems a better choice to you, then I can update it too.)